include/encap.h

```diff
@@ -6,6 +6,10 @@
 #include "dev.h"
 #include "pktinfo.h"
 
+#define PKT_PASS_TO_UPF 1
```

Comment:
what are the difference between PKT_PASS_TO_UPF and PKT_SENT? Does it mean to send pkt to APP or not?

Reply:
"PKT_PASS_TO_UPF" means that gtp5g cannot handle the packet (e.g., not GTPv1, not TPDU, or not an end marker) and passes the packet to a userspace program for handling.

"PKT_SENT" indicates that gtp5g has successfully handled the packet and forwarded it to the N6/N9 interface.

Reply:
rename to PKT_TO_APP and PKT_FORWARDED